spec/truefi2/scripts/FixedTermLoan.sh

```diff
@@ -0,0 +1,7 @@
+certoraRun contracts/truefi2/FixedTermLoan.sol \
+  contracts/truefi2/BorrowingMutex.sol \
+  contracts/common/UpgradeableERC20.sol:ERC20 \
+  --verify FixedTermLoan:spec/truefi2/FixedTermLoan.spec \
+  --link FixedTermLoan:token=ERC20 \
+  --short_output \
+  --optimistic_loop
```

Comment:
You may notice that `--rule_sanity` is absent here. That's because sanity checks are giving a false-negative error on a fallback function. There is a task to fix that in a followup PR.